server/db/index.js

```diff
@@ -4,7 +4,9 @@ require('dotenv').config();
 const defaultUri = 'mongodb+srv://' + process.env.USERNAME + ':' + process.env.PASSWORD
   + '@lobstericecream-trflp.gcp.mongodb.net/home?retryWrites=true&w=majority';
 const options = {
-  useNewUrlParser: true, useUnifiedTopology: true
+  useNewUrlParser: true, 
+  useUnifiedTopology: true,
+  useFindAndModify: false,
```

Comment:
Did you add this because of the findOneAndUpdate deprecation warning?

Reply:
yes, specifically for findByIdandRemove